### PR TITLE
Reworked for simplicity and consistency.  Removed frequency control.

### PR DESCRIPTION
**Pull request by jtchitty**

### CODE REVIEW

File: buzzerComponent/buzzer.c [hunk 1]
```diff
@@ -11,278 +25,289 @@
...
+static void SetBuzzerHz
+(
+    uint freq    ///< The buzzer frequency, in Hz. Use BUZZER_OFF_FREQ to stop the buzzer.
+)
 {
-    if (freqFp == NULL)
+    // Path to the RTC CLKOUT control file in sysfs.
+    static const char BuzzerFreqPath[] = "/sys/bus/i2c/drivers/rtc-pcf85063/8-0051/clkout_freq";
+
+    static FILE *FreqFile = NULL;
```
Comment (dpfrey) [inline buzzerComponent/buzzer.c:68]: Is variable capitalization related to the lifetime or scope of the variable?

Comment (jtchitty) [reply buzzerComponent/buzzer.c:68]: I'm following this:
https://docs.legato.io/latest/ccodingStdsFormat.html#codingFormatNamingStaticVar

Comment (dpfrey) [reply buzzerComponent/buzzer.c:68]: Disregard the above comment.  It's clear after reading this: https://docs.legato.io/latest/ccodingStdsFormat.html#codingFormatNamingStackVar

File: buzzerComponent/buzzer.c [hunk 2]
```diff
@@ -11,278 +25,289 @@
...
+    le_timer_SetMsInterval(Timer, PeriodMs * DutyCycleOnPercent / 100);
+    le_timer_Start(Timer);
+}
+
+//--------------------------------------------------------------------------------------------------
+/**
+ * Disable the buzzer, immediately stopping it, even if it's in the middle of a duty cycle.
+ */
+//--------------------------------------------------------------------------------------------------
+static void StopCycle()
```
Comment (dpfrey) [inline buzzerComponent/buzzer.c]: I prefer that all zero argument functions explicitly specify `void` as their argument list.  Technically there's no difference in the case of function definitions, but often times definitions get copied into declarations later on.

**Review by dpfrey — CHANGES_REQUESTED** [review]
Looks good.  Just one small change requested.

File: buzzerComponent/buzzer.c (hunk 2 above)
Comment (jtchitty) [reply buzzerComponent/buzzer.c]: I tend to agree.  I'll change this.

**Review by dpfrey — APPROVED** [review]

**Pull request merged**